packages/rtm-api/package.json

```
@@ -41,8 +41,8 @@
     "test": "npm run build && echo \"Tests are not implemented.\" && exit 0"
   },
   "dependencies": {
-    "@slack/logger": "^1.0.0",
-    "@slack/web-api": "^5.1.0",
+    "@slack/logger": "^2.0.0",
```

```suggestion
    "@slack/logger": ">=1.0.0 <3.0.0",
```

same reasoning as above ☝️